README.md

```diff
@@ -59,6 +59,7 @@ All options are optional, none are required.
 | ---- | ----------- | ------- |
 | **root** | Root path to resolve the include from | the file's path. |
 | **addDependencyTo** | Object with addDependency() method, taking file path as an argument. Called whenever a file is included. | |
+| **alias**| Object with alias mappings, { key: path } | |
```

Comment:
Maybe slightly more explicit with the example. Also how does it resolve the path? Absolute, relative, relative to ?

> Object with alias mappings, each key is your reference and the corresponding value is the path include's will use to resolve, { button: 'node_modules/button-package/button.html' }